readme.didact.md

```diff
@@ -160,14 +160,14 @@ oc label namespace camel-knative knative-eventing-injection=enabled
 
 ## 3. Push Bitcoin market data to the mesh
 
-We'll create a ([MarketSource.java](didact://?commandId=vscode.open&projectFilePath=MarketSource.java "Opens the file"){.didact}) integration
+We'll create a ([market-source.yaml](didact://?commandId=vscode.open&projectFilePath=market-source.yaml "Opens the file"){.didact}) integration, using Camel **YAML DSL**,
 with the role of taking live data from the Bitcoin market and pushing it to the event mesh, using the `market.btc.usdt` event type:
 
 
 ```
-kamel run MarketSource.java --logs
+kamel run market-source.yaml -d camel-jackson --logs
```

Comment:
what about adding the dependency in the modeline? it will simplify the command to launch for users and allows to have a working standalone files.

Reply:
I did not use modeline because this QS is also available for testing with TP1, which does not support modeline options